client/src/custom.scss

```diff
@@ -0,0 +1,67 @@
+// Override default variables before the import
+$primary: #7866D3;
+$offcanvas-vertical-height: 75vh;
+
+.border-dashed {
+  border-style: dashed !important;
+}
+
+.subscriptionListCard {
+  position: relative !important;
+}
+
+.innerButtonEdit {
+  position: absolute !important;
+  right: 0 !important;
+  top: 0 !important;
+}
+
+.innerButtonDelete {
+  position: absolute !important;
+  right: 0 !important;
+  bottom: 0 !important;
+}
```

Comment:
I think instead of absolutely positioning those icon buttons, we can achieve the same effect just using the flexbox classes in bootstrap

if you have a parent1 container with a child left container (name, due date, amount) and a child right container (edit and delete icons), then the parent just needs `d-flex justify-content-between` to have them be on opposite sides of the parent container. then within the child right container, if you vertically align them with `d-flex flex-column` it should stack vertically, then you can justify them either between or around or however you think looks best